src/sentry/api/endpoints/organization_events.py

```diff
@@ -126,8 +126,6 @@ def get(self, request, organization):
             except NoProjects:
                 return Response([])
 
-            params["organization_id"] = organization.id
```

Comment:
Why did you remove this? It seems like you're using it when you call `parse_release`

Reply:
Aiming for consistency where we set stuff on the `params` dict, I moved this into the `get_filter_params` [here](https://github.com/getsentry/sentry/pull/18568/files#diff-f348b3a3d06653db5b246ec567d39dc4R286)